File: app/src/main/java/com/decode/tumblr/model/Photo.java
package com.decode.tumblr.model;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class Photo {

    @SerializedName("caption")
    public String caption;

    @SerializedName("alt_sizes")
    public List<AltSize> altSizes = null;

}

File: app/src/main/java/com/decode/tumblr/data/model/Photo.kt
package com.decode.tumblr.data.model

import com.google.gson.annotations.SerializedName

data class Photo(

        @SerializedName("caption")
        var caption: String? = null,

        @SerializedName("alt_sizes")
        var altSizes: List<AltSize>? = null

)
